packages/core/src/Modules/Cashier/Containers/cashier.jsx

```diff
@@ -12,13 +12,14 @@ import {
 import { routes, isCryptocurrency, getSelectedRoute, isMobile, isTouchDevice } from '@deriv/shared';
 import { localize, Localize } from '@deriv/translations';
 
+import { WS } from 'Services';
 import { connect } from 'Stores/connect';
 import 'Sass/app/modules/cashier.scss';
 
 const el_landscape_blocker = document.getElementById('landscape_blocker');
 
 class Cashier extends React.Component {
-    state = { device_height: window.innerHeight };
+    state = { device_height: window.innerHeight, is_restricted: true };
```

Comment:
Can we use a name that's more descriptive of what the restriction is on? This would be useful as we have multiple configs/pages in Cashier.

```suggestion
    state = { device_height: window.innerHeight, is_p2p_restricted: true };
```

Reply:
This is done 👍